dotnet/ios/ios13/SupportingMultipleWindowsOniPad/Gallery/View Controllers/PhotoDetailViewController/PhotoDetailViewController.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+A view controller that contains an image view to show a particular photo.
+*/
+
+namespace Gallery;
+public partial class PhotoDetailViewController : UIViewController {
+	public Photo? Photo { get; set; }
+
+	protected PhotoDetailViewController (IntPtr handle) : base (handle)
+	{
+	}
+
+	public override void ViewDidLoad ()
+	{
+		base.ViewDidLoad ();
+		// Perform any additional setup after loading the view, typically from a nib.
+	}
+
+	public override void ViewWillAppear (bool animated)
+	{
+		base.ViewWillAppear (animated);
+
+		if (Photo is not null && Photo.Name is not null)
```

Comment:
You can simplify this, for example with:
```suggestion
		if (Photo?.Name is not null)
```
Why, because if Photo is null we will get null, else we will get Name, which might or not be null

Reply:
Ah I like this trick haha